README.md

```diff
@@ -44,7 +44,7 @@ your proposal!
 
 `toppra` was originally developed by [Hung
 Pham](https://hungpham2511.github.com/) (Eureka Robotics, former CRI
-Group) and [Phạm Quang Cường](https://personal.ntu.edu.sg/cuong/)
+Group) and [Phạm Quang Cưong](https://personal.ntu.edu.sg/cuong/)
```

Comment:
Do these characters break anything for you? These are valid Vietnamese characters.

Reply:
They are raising errors on Windows for English users.